README.md

```diff
@@ -27,6 +29,8 @@ It will listen on <http://127.0.0.1:3030> by default.
 
 Note: If you have issues with `npm install` please update to npm 2.0+ using `npm install -g npm@2` 
 ([Issue #1594](https://github.com/mozilla/fxa-content-server/issues/1594))
+Note: If you have issues with bigint (gmp.h not found) that stops pm2 from being installed, install pm2
```

Comment:
Hmm, should this whole section basically just be "Go to fxa-local-dev and run npm install then pm2 start servers.json"? And maybe a note to restart the server with pm2 to pick up any server changes.

@vladikoff What do you think? There's not much value in running this server in isolation– should we always suggest pm2?